recipes/install.rb

```diff
@@ -282,7 +282,7 @@
         'jdbc_connection_pools' => {
           'hopsworksPool' => {
             'config' => {
-              'datasourceclassname' => 'com.mysql.jdbc.jdbc2.optional.MysqlDataSource',
+              'datasourceclassname' => 'com.mysql.cj.jdbc.MysqlDataSource',
```

Comment:
This configuration is ignored during upgrades. That's why we have the create also in the `recipes/default.rb`.
So if we need to apply modifications to the `hopsworksPool` we need to drop it and re-create it in the `recipes/default.rb` - as I mentioned above.

Reply:
fxd